setup.py

```diff
@@ -49,25 +49,6 @@ def get_all_extras() -> Dict:
 all_extras = get_all_extras()
 
 
-def _get_docker_dependency() -> str:
-    """
-    Get the `docker-py` dependency, according to the underlying platform.
-
-    If on Windows, we need to install the cutting-edge package version of `docker` directly from the GitHub repository;
-    as the PyPI distribution of `docker` causes the following conflict:
-
-        docker 5.0.3 depends on pywin32==227; sys_platform == "win32"
-        open-aea[all] 1.20.0 depends on pywin32==304
-
-    Instead, at commit docker/docker-py@3f0095a, `setup.py` has an extra that forces pywin32>=304.
-    """
-    return (
-        "docker==5.0.3"
-        if platform.system() != "Windows"
-        else "docker @ git+https://github.com/docker/docker-py.git@3f0095a7c1966c521652314e524ff362c24ff58c"
-    )
```

Comment:
Not necessary since we updated `docker`.

Reply:
Same question as above